### PR TITLE
SDI-4: ✨ Add extra fields to visit with visitors endpoint

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 1]
```diff
@@ -499,6 +491,15 @@ private void validateActivity(final Long activityId) {
         return new PageImpl<>(visitsWithVisitors, pageable, visits.getTotalElements());
     }
 
+    private RelationshipType getRelationshipType(final VisitInformationFilter filter, final Long personId) {
+        if (personId == null) return null;
+        return offenderContactPersonsRepository.findAllByPersonIdAndOffenderBooking_BookingId(personId, filter.getBookingId())
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:496]: I think you could have a repository method like `findFirstBy....OrderByLastUpdated` or something like that

**Review by andymarke — APPROVED** [review]
LGTM

**Review by hyderkhanmoj — APPROVED** [review]
LGTM

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**